filters/ThirdParty/filter_246_AdblockPolska/metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "filterId": 117,
+  "filterId": 246,
   "name": "Adblock Polska",
   "description": "polski supplement for EasyList",
   "timeAdded": 1404115015843,
   "homepage": "https://www.facebook.com/adblockpl",
   "expires": "2 days",
-  "displayNumber": 107,
-  "groupId": 2,
+  "displayNumber": 246,
+  "groupId": 3,
```

Comment:
I don't think we should change the group.

We'' rework the filters groups soon, the current state is not good anyway.